gradle/components/klime-genmodel.gradle

```diff
@@ -0,0 +1,3 @@
+dependencies {
```

Comment:
imho should also be included in main build.gradle (published projects, java projects...)

Reply:
What should be included exactly? I followed the same pattern as the xgboost genmodel extension.